### ISSUE TITLE
Accessibility attributes: use ngAria or manually

### ISSUE THREAD

**Issue by tiltec** [yunity/foodsaving-frontend#108]
aria attributes should be set for screenreader support. Angular provides some magic:
https://docs.angularjs.org/guide/accessibility

Check if this is really good or needs too much workarounds (e.g. tabindex). See also #107

**Comment by tiltec** [comment]
I'm closing this to focus on features and code improvements first. Accessibility is not to be neglected, but can be improved at a later point.
